code-scanning/properties/synopsys-actions.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Synopsys Action",
+    "creator": "Synopsys",
+    "description": "The Synopsys GitHub Action allows you to configure your pipeline to run Synopsys security testing and take action on the security results",
+    "iconName": "synopsys-action",
```

Comment:
This icon is missing from the [icons](https://github.com/actions/starter-workflows/tree/main/icons) directory.

Reply:
Added logo into icons dir.